From 3fed80daf1dcb0b5d20e623d27228726c735e138 Mon Sep 17 00:00:00 2001
From: Bernd Kuhls <bernd.kuhls@t-online.de>
Date: Sun, 27 Sep 2015 19:09:47 +0200
Subject: [PATCH 1/1] xf86drm.c: Include limits.h to fix build error on Solaris
 and with musl

musl's strict implementation requires #include <limits.h> for PATH_MAX.

Patch suggested by evgeny for Solaris:
https://bugs.freedesktop.org/show_bug.cgi?id=92082

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>
---
 xf86drm.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/xf86drm.c b/xf86drm.c
index a9f5c29..ec985eb 100644
--- a/xf86drm.c
+++ b/xf86drm.c
@@ -53,6 +53,7 @@
 #include <sys/ioctl.h>
 #include <sys/time.h>
 #include <stdarg.h>
+#include <limits.h>
 #ifdef HAVE_SYS_MKDEV_H
 # include <sys/mkdev.h> /* defines major(), minor(), and makedev() on Solaris */
 #endif
-- 
2.5.3

